docs/projects/tcet-linux-website/frontend-components/footer.mdx

```diff
@@ -0,0 +1,136 @@
+---
+title: Footer
+id: footer
+description: Footer Component in Tcet Linux Website
+sidebar_label: Footer
+keywords: [Footer, social media,  ]
+---
+
+The **Footer** elelment of [**TCET Linux Website**](https://linux.tcetmumbai.in/) which is located at the bottom of the website. It serves as a container for information that is relevant to the entire page. It acts as a `Navigation` and `Contact ` guide for users. In the Footer element, We have included links to [**Documentation**](https://opensource.tcetmumbai.in/docs/category/projects/), [**Blogs**](https://opensource.tcetmumbai.in/blog/), **Live projects**, **Teams**, **Copyright Information**, [**Email address**](mailto:linux@tcetmumbai.in) for contact purposes with **TCET LINUX**, and icons linking to the Organization's presence on various social media platforms like **LinkedIn**, **YouTube**, **Twitter**, **Instagram**, and **GitHub**. We created this enhanced footer that helps users easily access relevant resources and connect with the **TCET OPENSOURCE ORGANISATION** through multiple channels.
```

Comment:
- Include Links for Social Medias as well
- Organization ka spelling wrong hai and instead of giving it uppercase put it in normal case with bold styling.